runtime/gc_base/RootScanner.cpp

```diff
@@ -81,6 +81,7 @@ MM_RootScanner::doClassLoader(J9ClassLoader *classLoader)
 void
 MM_RootScanner::scanModularityObjects(J9ClassLoader * classLoader)
 {
+#if JAVA_SPEC_VERSION > 8
```

Comment:
Does `classLoader->moduleHashTable` exist when JAVA_SPEC_VERSION == 8?  If so I would rather change the code to look like:
```
if (NULL != classLoader->moduleHashTable) {
....
}
```
instead of a compile time check

Reply:
`classLoader->moduleHashTable` does exist when `JAVA_SPEC_VERSION == 8` for now.
Will change the compile check to runtime check as per suggested.